src/test/kotlin/CohortProperties.kt

```diff
@@ -19,7 +20,8 @@ class CohortProperties(
                 userName = properties.getProperty("user.name")!!,
                 userPassword = properties.getProperty("user.password")!!,
                 cohort = properties.getProperty("cohort")!!,
-                jiraType = properties.getProperty("jira.type")
+                jiraType = properties.getProperty("jira.type"),
+                configProperties = properties.getProperty("configProperties")
```

Comment:
Can we initialise `configProperties` here immediately with `ConfigProperties.load(properties.getProperty(".."))` and hold the configuration object instead of the name of the properties file. I think this would simplify logic below.